account.go

```diff
@@ -67,3 +101,14 @@ func (c *Client) GetAccount(ctx context.Context) (*Account, error) {
 
 	return response, nil
 }
+
+// UpdateAccount updates the Account
+func (c *Client) UpdateAccount(ctx context.Context, opts AccountUpdateOptions) (*Account, error) {
+	e := "account"
+	response, err := doPUTRequest[Account](ctx, c, e, opts)
+	if err != nil {
+		return nil, err
+	}
+
+	return response, nil
```

Comment:
```suggestion
	return doPUTRequest[Account](ctx, c, e, opts)
```
I think we can simplify this to:

Reply:
Yeah good call. Fixed this in all the files in this PR!